compendium/DeclarativeServices/src/ConfigurationListenerImpl.cpp

```diff
@@ -80,17 +80,18 @@ namespace cppmicroservices
                     }
                     auto properties = cppmicroservices::AnyMap(AnyMap::UNORDERED_MAP_CASEINSENSITIVE_KEYS);
                     auto type = event.getType();
-
+                    auto changeCount_ = -1;
```

Comment:
Identifier names that end in "_" are usually fields in a class, not local variables.